### PR TITLE
Fix Issue 2179 - Installing Playwright on Ubuntu 24 LTS and using Python 3.12

### PR DESCRIPTION
**Pull request by emilienbev**
# Description

Upgrades Playwright's version to 1.48.0 which supports Ubuntu 24 LTS, and adds Python 3.12 minor version support to `main.py`.

# Issue Fixes

Fixes Issue https://github.com/elebumm/RedditVideoMakerBot/issues/2179

# Checklist:

- [x] I am pushing changes to the **develop** branch
- [x] I am using the recommended development environment
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have formatted and linted my code using python-black and pylint
- [x] I have cleaned up unnecessary files
- [x] My changes generate no new warnings
- [x] My changes follow the existing code-style
- [x] My changes are relevant to the project

# Any other information (e.g how to test the changes)

Running a fresh installation on Ubuntu 24.04 LTS will now work, and running the program functions correctly.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -20,8 +20,8 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           stale-issue-message: 'This issue is stale because it has been open 7 days with no activity. Remove stale label or comment, or this will be closed in 10 days.'
           close-issue-message: 'Issue closed due to being stale. Please reopen if issue persists in latest version.'
-          days-before-stale: 6
```
Comment (JasonLovesDoggo) [inline .github/workflows/stale.yml:23]: lgtm, just rebase your pr so that these changes are not reverted

**Comment by disarticulate** [comment]
woud love to upgrade one of my containers to use latest 24 LTS.

**Comment by emilienbev** [comment]
Thanks for reviewing! Should be updated now.

**Comment by JasonLovesDoggo** [comment]
LGTM thanks!

**Pull request merged**